controllers/news.controllers.js

```diff
@@ -1,7 +1,16 @@
-const { selectTopics } = require("../models/news.models")
+const { selectTopics, selectArticles, selectArticle } = require("../models/news.models")
 
 exports.getTopics = (req, res, next) => selectTopics().then((topics) => {
-        return res.status(200).send({topics})
+        res.status(200).send({ topics })
     }).catch((err) => {
         next(err)
     })
+
+exports.getArticles = (req, res, next) => selectArticles().then( (articles) => {
+	res.status(200).send({ articles })
+}).catch( (err) => next(err))
+
+exports.getArticle = (req, res, next) => {
+	selectArticle(req.params.article_id)
+	.then( (resArticle) => {res.status(200).send(resArticle)}).catch((err) => next(err))
```

Comment:
it looks like you've got `.send(resArticle)})` it's missing an opening `{` curly bracket inside the `( )`